x/mint/types/params.go

```diff
@@ -73,15 +73,19 @@ func (p *Params) ParamSetPairs() paramtypes.ParamSetPairs {
 }
 
 func validateMintDenom(i interface{}) error {
-	v, ok := i.(string)
+	denomString, ok := i.(string)
+	denomTrimed := strings.TrimSpace(denomString)
+
 	if !ok {
 		return fmt.Errorf("invalid parameter type: %T", i)
 	}
-
-	if strings.TrimSpace(v) == "" {
+	if denomTrimed == "" {
 		return errors.New("mint denom cannot be blank")
 	}
-	if err := sdk.ValidateDenom(v); err != nil {
+	if denomTrimed != "usei" {
```

Comment:
nit - is there a variable we can use that specifies base denom? maybe
```
x/dex/types/keys.go:155:        DefaultPriceDenom = "usei"
```

Reply:
gotcha, using the default bond denom